client/me/purchases/manage-purchase/index.jsx

```diff
@@ -538,6 +546,7 @@ class ManagePurchase extends Component {
 					<PurchaseMeta
 						purchaseId={ false }
 						siteSlug={ siteSlug }
+						hasLoadedPurchasesFromServer={ hasLoadedPurchasesFromServer }
 						getManagePurchaseUrlFor={ getManagePurchaseUrlFor }
 						getAddCardDetailsPathFor={ getAddCardDetailsPathFor }
```

Comment:
I know this isn't part of your diff, but from what I can tell, `getAddCardDetailsPathFor` is not used by `PurchaseMeta`. I wonder why we added that... is it a mistake here or in PurchaseMeta?

Reply:
Good catch - I can't find where that prop was ever used. It looks like I'm the one that introduced it in #46069, so it was probably a mistake. :)